### ISSUE TITLE
1.13 /mh whitelist add error

### ISSUE THREAD

**Issue by Nesseley** [Rocologo/MobHunting#421]
this error accours when using /mh whitelist add
https://pastebin.com/UK3Nq7zY

**Comment by Rocologo** [comment]
Which version of ProtocolLib does you use?

**Comment by Nesseley** [comment]
I do not use it, as the spigot page states it's optional.

**Comment by Rocologo** [comment]
Thats correct, its optional, but it seems that have not tested whitelist without protocollib installed.

Im not able to fix this to night, so if this is urgent, then you should install protocollib until i have fixed this bug

**Comment by Rocologo** [comment]
Please test build 691

https://fractal.lindegaard.one:8181/job/MobHunting/691/

**Comment by Rocologo** [comment]
Thank you.
